common/supplemental/attributeValueValidity.xml

```diff
@@ -12,7 +12,7 @@
 			<!-- The following are in $language but not in comon/main or in modern coverage: cu lij prg vo -->
 			<variable id='$language' type='choice'>
 				af agq ak am ann ar as asa ast az
-				bas be bem bez bg bm bn bo br brx bs
+				bas be bem bez bg bho bm bn bo br brx bs
```

Comment:
Actually we do not want to add this. We already have `bho` in the $languageExceptions section of this file, so vetters are already being asked for its display name in different languages. The codes in the $language section are generally only for locales we have in common/main/.